interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -123,7 +123,7 @@
 <body bgcolor="{$STYLE.BGCOLOR2}">
 <p><table><tr><td><span class="title">{xl t="Vitals"}</span></td><td>&nbsp;&nbsp;&nbsp;<a href="../summary/demographics.php" class="readonly css_button_small" onclick="top.restoreSession()"> <span>{xl t="View Patient"}</span></a></td></tr></table></p>
 <form name="vitals" method="post" action="{$FORM_ACTION}/interface/forms/vitals/save.php" onSubmit="return vitalsFormSubmitted()">
-<div id="chart"></div>
+<div id="chart" style="direction:ltr;"></div>
```

Comment:
If in `core.css` you add 

```CSS
#chart {
    direction:ltr;
}
```

You wouldn't have to repeat yourself. Stay DRY!